packages/carousel/src/react/__stories__/index.story.js

```diff
@@ -181,20 +183,77 @@ Object.values(Carousel.sizes).forEach(size => {
   ))
 })
 
-storiesOf('Carousel/with ActionMenu', module).add('positioned child', () => (
-  <div style={{ border: '1px solid red', maxWidth: 400, padding: 10 }}>
-    <Carousel size={Carousel.sizes.wide}>
-      <MockItem>
-        <ActionMenu style={{ left: 20, top: 20 }}>
-          {new Array(8).fill(null).map((_, index) => (
-            <ActionMenu.Item key={index}>item: {index}</ActionMenu.Item>
-          ))}
-        </ActionMenu>
-      </MockItem>
+storiesOf('Carousel/with ActionMenu', module)
+  .add('positioned child', () => (
+    <div style={{ border: '1px solid red', maxWidth: 400, padding: 10 }}>
+      <Carousel size={Carousel.sizes.wide}>
+        <MockItem>
+          <ActionMenu style={{ left: 20, top: 20 }}>
+            {new Array(8).fill(null).map((_, index) => (
+              <ActionMenu.Item key={index}>item: {index}</ActionMenu.Item>
+            ))}
+          </ActionMenu>
+        </MockItem>
 
-      {new Array(13).fill(null).map((_, index) => (
-        <MockItem key={index} />
-      ))}
-    </Carousel>
-  </div>
-))
+        {new Array(13).fill(null).map((_, index) => (
+          <MockItem key={index} />
+        ))}
+      </Carousel>
+    </div>
+  ))
+  .add('in portal', () => {
+    function PortalStory() {
+      const [isOpen, setOpen] = React.useState(false)
+      return (
+        <div style={{ border: '1px solid red', maxWidth: 600, padding: 10 }}>
+          <Carousel
+            size={Carousel.sizes.wide}
+            controls={
+              <Carousel.Controls>
+                <Carousel.Control
+                  direction={Carousel.Control.directions.prev}
+                  onClick={_ => setOpen(false)}
+                />
+                <Carousel.Control
+                  direction={Carousel.Control.directions.next}
+                  onClick={_ => setOpen(false)}
+                />
+              </Carousel.Controls>
+            }
+          >
+            <MockCard
+              actionBarVisible
+              actionBar={[
+                <BelowRight
+                  inNode={document.body}
+                  when={isOpen}
+                  show={
+                    <ActionMenu>
+                      {new Array(8).fill(null).map((_, index) => (
+                        <ActionMenu.Item key={index}>
+                          item: {index}
+                        </ActionMenu.Item>
+                      ))}
+                    </ActionMenu>
+                  }
+                  key="a"
```

Comment:
this should be unique

Reply:
unique w/in the list, right?  And it is, I think.